fleet_adapter_mir/README.md

```diff
@@ -41,8 +41,70 @@ Alternatively, if you want to run everything with full capabilities, (though not
 ros2 run fleet_adapter_mir fleet_adapter_mir -c mir_config.yaml -n nav_graph.yaml
 ```
 
+If you have not configured the necessary RMF missions on the MiR, you may parse the relevant JSON filepaths when launching the fleet adapter node. On startup, the fleet adapter will create these missions via MiR REST API
+
+```bash
+ros2 run fleet_adapter_mir fleet_adapter_mir -c mir_config.yaml -n nav_graph.yaml -a ../missions/rmf_missions.json
+```
+
 
 
 ### Configuration
 
-An example configuration file, `mir_config.yaml` has been provided. It has been generously commented, and in the cases where it has not, the parameter names are self-explanatory enough.
\ No newline at end of file
+An example configuration file, `mir_config.yaml` has been provided. It has been generously commented, and in the cases where it has not, the parameter names are self-explanatory enough.
```

Comment:
From reading the Python code implementation, it looks like the localization mission in particular has quite a few requirements around how it needs to be constructed which a user would only know from reading and understanding the Python code or from repeatedly trying to run the fleet adapter and looking at what assertions fail.

I think it would be best if we can document a rough sketch of what each basic mission type is supposed to look like. There should be enough information in the documentation for users to recreate the missions themselves in the MiR UI without looking at our Python code.

You could include that documentation here in this README, but it may be cleaner to create a new `docs` folder and put a file in there that the README can link to.

> in the cases where it has not, the parameter names are self-explanatory enough

The wording here may come off a bit aggressive. I recommend tweaking this to "in the cases where it has not, the parameter names are meant to be self-explanatory". Otherwise it may sound like we're blaming the reader if they don't understand the parameter names, but things that seem self-evident to us will not be self-evident to everyone.

Reply:
I'm noticing that some of the missions are created automatically, so we should also document what missions will be created automatically and what those will look like to the user so they don't get alarmed when they see new missions pop up in the MiR after using this fleet adapter.

Reply:
I agree the mission organization is a little messy right now, I'll take the `docs` folder suggestion and update accordingly.

> The wording here may come off a bit aggressive. I recommend tweaking this to "in the cases where it has not, the parameter names are meant to be self-explanatory". Otherwise it may sound like we're blaming the reader if they don't understand the parameter names, but things that seem self-evident to us will not be self-evident to everyone.

Looks like this line is from the current package README, but I'll take note and modify the language. Thanks for highlighting!